tests/infer/test_elbo_mapdata.py

```diff
@@ -17,9 +17,12 @@
 
 @pytest.mark.stage("integration", "integration_batch_1")
 @pytest.mark.init(rng_seed=161)
-@pytest.mark.parametrize("batch_size", [3, 8, None])
-@pytest.mark.parametrize("map_type", ["plate", "iplate", "range"])
-def test_elbo_mapdata(batch_size, map_type):
+@pytest.mark.parametrize("map_type, batch_size, params", [("iplate", 3, (7000, 0.0008)), ("iplate", 8, (100, 0.018)),
```

Comment:
I think what @martinjankowiak is suggesting is to write this as
@pytest.mark.parametrize("map_type, batch_size, n_steps, lr", [
    ("iplate", 3, 7000, 0.0008),
    ("iplate", 8, 100, 0.018),
    ...
])